### PR TITLE
Give a friendly preview to the integration documentation

### PR DESCRIPTION
**Pull request by brucellino**
# Summary 
This PR describes the journey and tasks before the reader who wants to integrate their site.

----

**Related issue :** #15

### CODE REVIEW

File: source/integration.rst [hunk 1]
```diff
@@ -1,7 +1,20 @@
 Integration
 -----------
 
-You can follow dedicated integration guides for each cloud management framework:
+Integration of cloud stacks into EGI FedCloud follows a well-defined path, with certain steps which need to be taken, depending on the cloud stack in question. By integration here, we refer to the proper interoperation with EGI infrastructure services such as accounting, monitoring, authentication and authorisation, _etc_. These configurationsmake your site discoverable and usable by the communities you wish to support, and allow EGI to support you in operational and technical matters.
```
Comment (gwarf) [inline source/integration.rst]: Missing space

File: source/integration.rst [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+
+Integration of these services implies specific configuration actions which you need to take on your site. These aim to be unintrusive and are mostly to facilitate access to your site by the communities you wish to support, without interfering with normal operations. This can be summarised essentially as :
+
+  1. Network configuration 
+  1. Permissions configuraiton
```
Comment (gwarf) [inline source/integration.rst]: Typi

File: source/integration.rst (hunk 1 above)
Comment (brucellino) [reply source/integration.rst]: I can live with one sentence per line, thanks for the suggestion.

File: source/openstack.rst [hunk 3]
```diff
@@ -1,3 +1,6 @@
+.. toctree::
+.. sectnum::
+
```
Comment (enolfc) [inline source/openstack.rst]: Should we add this to the opennebula.rst?

**Comment by enolfc** [comment]
On long lines: I started this by copy-pasting from the wiki, so it's full of long lines, I don't like them either but I didn't have the time/willingness to do nice wrapping. Is there any automatic way of doing that?

File: source/integration.rst [hunk 4]
```diff
@@ -1,7 +1,23 @@
...
 
-You can follow dedicated integration guides for each cloud management framework:
+Integration of cloud stacks into EGI FedCloud follows a well-defined path, with certain steps which need to be taken, depending on the cloud stack in question.
+By integration here, we refer to the proper interoperation with EGI infrastructure services such as accounting, monitoring, authentication and authorisation, _etc_.
+These configurationsmake your site discoverable and usable by the communities you wish to support, and allow EGI to support you in operational and technical matters.
+
+Integration of these services implies specific configuration actions which you need to take on your site.
+These aim to be unintrusive and are mostly to facilitate access to your site by the communities you wish to support, without interfering with normal operations. This can be summarised essentially as :
+
+  1. Network configuration
```
[the remark this answers is not in the document]
Comment (brucellino) [reply source/integration.rst]: I was referring to the list of necessary ports. Perhaps should be "firewall configuration"?

File: source/openstack.rst (hunk 3 above)
Comment (brucellino) [reply source/openstack.rst]: will do.

File: source/integration.rst (hunk 2 above)
Comment (brucellino) [reply source/integration.rst]: > On long lines: I started this by copy-pasting from the wiki, so it's full of long lines, I don't like them either but I didn't have the time/willingness to do nice wrapping. Is there any automatic way of doing that?

Only thing I could find is https://github.com/mrmlnc/vscode-remark

**Comment by brucellino** [comment]
I think I have addressed the requests of the fine reviewers :nose:

**Comment by brucellino** [comment]
@gwarf are you happy with the updates?

**Comment by enolfc** [comment]
A kindly reminder to @gwarf for checking this one

**Pull request merged**